SpherePacking/ModularForms/eta.lean

```diff
@@ -17,7 +17,6 @@ noncomputable def η (z : ℂ) := cexp (2 * π * Complex.I * z / 24) * ∏' (n :
     (1 - cexp (2 * π * Complex.I * (n + 1) * z))
 
 
-/-this is being PRd-/
```

Comment:
Can we keep this comment around in some form?

Reply:
yeah it has been PRd, but we might need to update the file with the new version. I can sort it out though afterwards if thats easier